From 3d2fa6d4357b325d456c15fdaef3f22a99a3352f Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Tue, 23 Aug 2022 11:32:00 +0200
Subject: [PATCH 0380/1695] plat-stm32mp2: conf: enable OMM driver

This patch enables OMM driver that is in charge of configuring IOM IP.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 344d6712c7bd03453630ce56c4938488fea7917c plat-stm32mp2: conf: enable OMM driver]
Change-Id: I300c776d28e9926be5ccd3566fa8349d73c8b951
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 194a99505..b919ffd25 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -71,6 +71,7 @@ CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
 CFG_STM32_GPIO ?= y
 CFG_STM32_IAC ?= y
+CFG_STM32_OMM ?= y
 CFG_STM32_RIF ?= y
 CFG_STM32_RIFSC ?= y
 CFG_STM32_RISAB ?= y
-- 
2.47.2

